src/game/SpellEffects.cpp

```diff
@@ -4594,15 +4594,17 @@ void Spell::EffectSummonPet(SpellEffectIndex eff_idx)
 
     Pet* NewSummon = new Pet;
 
+    bool skip;
```

Comment:
yeah, this skip system is still wrong, hunters must NEVER get to line 4650 or they will always be returned in 4655, along with a false flagged error.